genie-web/src/main/java/com/netflix/genie/web/exceptions/checked/AgentLaunchException.java

```diff
@@ -0,0 +1,55 @@
+/*
+ *
+ *  Copyright 2019 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.web.exceptions.checked;
+
+/**
+ * An exception for when the server can't launch an agent for whatever reason.
+ *
+ * @author tgianos
+ * @since 4.0.0
+ */
+public class AgentLaunchException extends Exception {
```

Comment:
Adding a mandatory constructor argument with the array of executable and arguments could be nice to debug.

Reply:
Seems useful. I'll add it in a subsequent commit in a different PR